packages/xchain-client/src/BaseXChainClient.ts

```diff
@@ -18,6 +18,7 @@ import {
 } from './types'
 
 const MAINNET_THORNODE_API_BASE = 'https://thornode.thorchain.info/thorchain'
+const STAGENET_THORNODE_API_BASE = 'https://stagenet-thornode.ninerealms.com'
 const TESTNET_THORNODE_API_BASE = 'https://testnet.thornode.thorchain.info/thorchain'
```

Comment:
Ohter endpoints might be changed to `ninerealmscom` as well?

Reply:
I believe we have a mainnet api endpoint that we can upgrade to, but not a testnet. Double-checking.

Reply:
API bases updated for mainnet